src/views/Generator/TestOptions/LoadZones/LoadZones.tsx

```diff
@@ -0,0 +1,87 @@
+import { LoadZoneSchema } from '@/schemas/generator/v1/loadZone'
+import { useGeneratorStore } from '@/store/generator/useGeneratorStore'
+import { LoadZoneData } from '@/types/testOptions'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { Table, Text, Link as RadixLink, Button } from '@radix-ui/themes'
```

Comment:
`Table` from `@/components/Table` should have the same header style as the rest of the tables in the app. I should probably rename it to avoid confusion